From bc065596b4dd69fa36e2f8319b63fc4cd9e18529 Mon Sep 17 00:00:00 2001
From: Julien Boibessot <julien.boibessot@armadeus.com>
Date: Sat, 29 May 2010 10:47:04 +0000
Subject: ncftp: bump version

Due to recent 3.2.4 release, 3.2.2 download address has changed, so
I prefered bump version to 3.2.4 rather than changing NCFTP_SITE.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>
Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
Index: buildroot/package/ncftp/ncftp.mk
===================================================================
--- buildroot.orig/package/ncftp/ncftp.mk	2008-11-03 09:15:12.000000000 +0100
+++ buildroot/package/ncftp/ncftp.mk	2010-06-23 16:11:34.000000000 +0200
@@ -3,7 +3,7 @@
 # ncftp client
 #
 #############################################################
-NCFTP_VERSION:=3.2.2
+NCFTP_VERSION:=3.2.4
 NCFTP_SOURCE:=ncftp-$(NCFTP_VERSION)-src.tar.bz2
 NCFTP_SITE:=ftp://ftp.ncftp.com/ncftp
 NCFTP_DIR:=$(BUILD_DIR)/ncftp-$(NCFTP_VERSION)
